data_access_layer/basic_db_functions.py

```diff
@@ -36,3 +36,21 @@ def delete_number_from_db(number: int):
     except Exception as e:
         print(e)
         return e
+
+def check_and_add_allowed_topics():
+    mongo_client = gb.mongo_client
+    if mongo_client:
+        db = mongo_client["telegram_bot_data"]
+        collection = db['topics']
+        
+        existing_topics = {doc['name'] for doc in collection.find()}
+
+        missing_topics = ALLOWED_TOPICS - existing_topics
+
+    if missing_topics:
+        collection.insert_many([{"name": topic} for topic in missing_topics])
+        print(f"Inserted missing topics: {missing_topics}")
```

Comment:
I got this warning "Local variable 'missing_topics' might be referenced before assignment" you should fix this ( place the if else statement inside the upper if)